packages/e2e-tests/specs/editor/various/reusable-blocks.test.js

```diff
@@ -330,4 +332,43 @@ describe( 'Reusable blocks', () => {
 		// Check that we have two paragraph blocks on the page
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	it( 'will not break the editor if empty', async () => {
+		await insertReusableBlock( 'Awesome block' );
+
+		await visitAdminPage( 'edit.php', [ 'post_type=wp_block' ] );
+
+		const [ editButton ] = await page.$x(
+			`//a[contains(@aria-label, 'Awesome block')]`
+		);
+		await editButton.click();
+
+		await page.waitForNavigation();
+
+		// Give focus to the editor
+		await page.click( '.block-editor-writing-flow' );
+
+		// Move focus to the paragraph block
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Tab' );
+
+		// Delete the block, leaving the reusable block empty
+		await clickBlockToolbarButton( 'More options' );
+		const deleteButton = await page.waitForXPath(
+			'//button/span[text()="Remove block"]'
+		);
+		deleteButton.click();
+
+		// Save the reusable block
+		await page.click( '.editor-post-publish-button__button' );
+		await page.waitForXPath(
+			'//*[contains(@class, "components-snackbar")]/*[text()="Reusable Block updated."]'
+		);
+
+		await createNewPost();
+
+		await toggleGlobalBlockInserter();
+
+		expect( console ).not.toHaveErrored();
```

Comment:
This check is scoped only to the e2e spec. It isn't possible to catch errors on the page this way. We have a global verification for the page:

https://github.com/WordPress/gutenberg/blob/23a69e244f24218e01b9e24eb0252a9e403ff4f2/packages/e2e-tests/config/setup-test-framework.js#L99

Reply:
Hmmm, interesting. It seemed like it did catch the error when I tested. Or maybe it was just the error bubbling up. Maybe I missed something. I'll double-check. Thanks for pointing that out 🙇🏻

Reply:
Do you mean then this assertion is not needed at all?

Reply:
It would mean there is `console.error` call in the test code, not on the page. What error do you get when it isn't called?

Reply:
I double-checked _(by re-introducing the bug that caused the regression)_ and that expect does seem to work for the purposes of the test, see the output in the gist:

https://gist.github.com/fullofcaffeine/453e58fba78d902bc690e0b6994e7b31

It is true that it might be considered redudant because if I remove it, the test still fails because the error bubbles up (as described here) and it catched from within the console object:

https://gist.github.com/fullofcaffeine/ade1be595e7f1c5bd46ed5b652814934

That being said, I'd rather keep the expect there as it communicates what the test is not expecting.

So, it seems it's fine the way it is? Let me know if I missed something 🙇🏻